manifests/P/PlasmoApp/plasmo_voice/main.yaml

```diff
@@ -0,0 +1,23 @@
+manifestSpecVersion: "4.0"
+publisher: PlasmoApp
+iconUrls:
+  - https://cdn.modrinth.com/data/1bZhdhsH/icon.png
+  - https://raw.githubusercontent.com/plasmoapp/plasmo-voice/client-1.18-1.2.x/common/src/main/resources/assets/plasmo_voice/icon.png
+status: active
+updatedAlternatives: ~
+name: "Plasmo Voice"
+description: A proximity voice chat mod with audio positioning and lots of features.
+authors:
+  - name: kpids
+  - name: Apehum
+home: https://github.com/plasmoapp/plasmo-voice
+source: https://github.com/plasmoapp/plasmo-voice
+issues: https://github.com/UltimateBoomer/Resolution-Control/issues
+support: ~
```

Comment:
Under support, you can choose which option from `chats` you think is best. But when there's only one option, like in this case, just also paste that one here. (So paste the discord here too)